test/built-ins/Object/fromEntries/iterator-closed-for-throwing-entry-value-tostring.js

```diff
@@ -0,0 +1,68 @@
+// Copyright (C) 2018 Kevin Gibbons. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-object.fromentries
+description: Closes iterators when toString on a value throws.
```

Comment:
This one isn't valid; only the keys get toString'd (or, to be precise, toPropertyKey'd). The values are passed through without being touched.

Reply:
Whoops, yes, that would make sense. I will drop this.